src/store/student-store.js

```diff
@@ -34,7 +35,7 @@ store.registerHandler('CLEAR_ALL_DATA', () => {
 
 store.registerHandler('CHECKOUT_SUCCESS', () => {
     student.items = student.items.concat(CartStore.getItems());
-    store.emitChange();
```

Comment:
I don't think it should be there. But I put it back. It'll all be changed once I refactor some of the API logic into controllers anyway.